bi/pkg/cluster/kind/net.go

```diff
@@ -41,23 +43,164 @@ func GetMetalLBIPs(ctx context.Context) (string, error) {
 }
 
 // Given a Network such as 172.18.0.0/16
-// Return the bottom half of the network, such as 172.18.1.0/24
+// Return an available subnet for MetalLB, such as 172.18.1.0/24
 func split(ipNet *net.IPNet, count int) (*net.IPNet, error) {
+	ones, bits := ipNet.Mask.Size()
+	slog.Debug("Split network details", 
+		slog.String("baseNetwork", ipNet.String()),
+		slog.Int("maskOnes", ones),
+		slog.Int("maskBits", bits))
+	
 	shift := calculateShift(ipNet)
-	return cidr.Subnet(ipNet, shift, count+1)
```

Comment:
Notice how we never gave out the bottom half. That's because almost all network layouts need a gateway and they usually have .1 reserved for that.